core/domain/exp_domain_test.py

```diff
@@ -717,14 +717,16 @@ def test_validation(self):
         init_state.update_interaction_default_outcome(default_outcome.to_dict())
         exploration.validate()
 
-        init_state.update_interaction_solution({
-            'answer_is_exclusive': True,
-            'correct_answer': 'hello_world!',
-            'explanation': {
-                'content_id': 'solution',
-                'html': 'hello_world is a string'
-                }
-        })
+        solution = state_domain.Solution(
+            interaction_id=init_state.interaction.id,
```

Comment:
You shouldn't need to specify the argnames. The only exception to this is when a default arg is specified in the definition of the function being called.

Please fix, here and elsewhere.

Reply:
Hi @FlorinBalint -- one note, please don't resolve the github comments. Instead, say "Done" or similar if you've completed them. Thanks!